packages/taquito-utils/src/errors.ts

```diff
@@ -1,142 +1,66 @@
-export { InvalidHexStringError, InvalidContractAddressError, InvalidAddressError, InvalidBlockHashError } from '@taquito/core';
+import { ValidationError } from '@taquito/core';
 
```

Comment:
nickpicking, would be nice to not have blank line2 🙈

Reply:
i think i prefer this since one is an import and the other is an export